apiserver/pkg/util/cluster.go

```diff
@@ -534,6 +540,54 @@ func buildVols(apiVolumes []*api.Volume) []v1.Volume {
 			}
 			vols = append(vols, vol)
 		}
+		if rayVol.VolumeType == api.Volume_EPHEMERAL {
```

Comment:
I am not sure how to handle the server panic.  Perhaps create an error object and bubble all the way up Or assign a default value like 1Gi

```suggestion
		if rayVol.VolumeType == api.Volume_EPHEMERAL {
			if rayVol.Storage == "" {
			     / / rayVol.Storage = "1Gi"
		             return vols, errors.New("Storage is empty.")
			}
```

Reply:
Yes, this sounds like the right approach

Reply:
@tedhtchang, I think I fixed it